### DISCUSSION TITLE
What would you replace Oxford's PPE course with if you wanted a degree fit for future leaders?

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#487] [General]
### Oxford's PPE course is *the* degree for political leaders, at least in the UK

- PPE is a legendary course at Oxford University. It stands for **Philosophy, Politics and Economics**
- PPE has been a default **curriculum for leaders** in politics, those who run our societies - at least in the UK[^1]
- e.g. Rupert Murdoch is a PPE'er, as is Ed Miliband, David Cameron, Rishi Sunak and [many more](https://en.wikipedia.org/wiki/List_of_University_of_Oxford_people_with_PPE_degrees)
- PPE started at Oxford in 1920 to "upgrade" Greats[^2]
- Inspiration for courses at over 50 other universities include 3 Ivy Leaguers, Harvard, Tsinghua.

### PPE as it currently stands is a course of the past both in content and purpose

- It is a curriculum of "modernity", of the 19th/20th centuries.
- Its philosophy is classical, its core is now economics -- the essential social science of modernity, with its tendency to rational agent reductionism, free markets etc.

### What would be a curriculum for future leaders?

- What would be a **curriculum of the future** be for leaders? For people wanting to engage in politics and social change?
- A curriculum for what comes after modernity?

###  A first suggestion: Psychology, Politics and Ecology

A first suggestion (tongue in cheek 😜): **Psychology, Politics and Ecology**

- Psychology broadly interpreted as ontology i.e. including mindfulness, the transpersonal etc plus culturology i.e. the study of culture and its evolution. It's "integral" inner development: both personal *and* collective plus all lines of development - cognitive, emotional, spiritual etc
- Politics should still be there, but not conventional party politics, or dry political history. A theory of collective action, community building and paradigmatic social change
- Ecology: representing a richer, broader and more humble approach to social and environmental systems "design" (or preferably stewardship! Moving away from those delusional engineering attitudes towards a more skilful and accurate sense of gardening, curation, tending)

📣 Suggestions welcome -- bonus if you can fit the acronym! 📣

## Background 

Back in 2019/2020 we were looked for young graduates to contribute at Life Itself. We were lucky to have many (great) applicants. It was noticeable that several of them had done an Oxford degree called "PPE" or Philosophy, Politics and Economics.

Now, if you're not British you probably never heard of PPE. But in the UK it's legendary degree that "runs Britain" --  huge numbers of the UK's top politicos and pundits have done it including several of the last prime ministers and leaders.[^1]

You could see this as the exemplar curriculum for training the leaders of today -- and tomorrow (or at least training, the ambitious and conscientious). Now in some ways it's a pretty awesome: it's interdisciplinary (rare in the UK) and at least it has some philosophy in there (surely a good idea to have a bit of ethics and metaphysics in there for our future leaders).

But ... in many ways its content seem pretty outdated. And it doesn't seem to always be producing the leaders we need ...

I started to try to imagine a replacement. What would a PPE of the future be? Trying to keep the acronym a first idea was: Psychology, Politics and Ecology ...

## Notes

- Imagine you showed up and the first thing you did was an intensive course in mindfulness or went off to the Landmark forum
- More generally, if inner development were a core aspect of the degree both theoretically **and pratically** -- i.e. emphasis on praxis rather than just theory.
- Imagine if a core emphasis of the course were wisdom and ethics: if you are going to lead you need incredible ethics ...
- What would "wise" leaders look like?

[^1]: see e.g. https://www.theguardian.com/education/2017/feb/23/ppe-oxford-university-degree-that-rules-britain "Monday, 13 April 2015 was a typical day in modern British politics. An Oxford University graduate in philosophy, politics and economics (PPE), Ed Miliband, launched the Labour party’s general election manifesto. It was examined by the BBC’s political editor, Oxford PPE graduate Nick Robinson, by the BBC’s economics editor, Oxford PPE graduate Robert Peston, and by the director of the Institute for Fiscal Studies, Oxford PPE graduate Paul Johnson. It was criticised by the prime minister, Oxford PPE graduate David Cameron. It was defended by the Labour shadow chancellor, Oxford PPE graduate Ed Balls."

[^2]: Greats is Oxfords's previous classics oriented "leadership" degree - the one Boris Johnson did. Over 50% of UK's prime ministers were educated at Oxford (30 of 57 - vs just 16 at Cambridge).

**Comment by baouroux** [comment]
An amazing post, thanks for all these references and context setting @rufuspollock.

I am in contact with the Danish alternative entrepreneurship Masters school KaosPilot on a very similar topic, I'd love to bring them into this discussion https://www.kaospilot.dk/. They've been going for over 30 years, and have tried to bridge the academic with entrepreneurial direct actions.

I love the Psychology, Politics and Ecology descriptions and moving these along a line of innovation and new framing.

Without yet formulating a polished contribution, I'm drawn to using the Transcendentals to inform the design - what are the fundamental contrasting truths that the human condition is called to "coordinate" - i.e. The Good, True and Beautiful...The Morals, Sciences and Arts...

Just a bit of fun to connect some dots, your new PPE matches I propose is this possible sequence:
Psychology = the Beautiful (Arts) = 1st person practices (incl ego development)
Politics = the Good (Morals) = 2nd person practices (incl cultural anthropology)
Ecology = the True (Sciences) = 3rd person practices (incl systems complexity)

Of course, you and I have mentioned William Irwin Thompson's amazing work with the Ross School curriculum (https://spiral.ross.org/spiral/#/) - see #387

**Reply by rufuspollock** [reply]
> I am in contact with the Danish alternative entrepreneurship Masters school KaosPilot on a very similar topic, I'd love to bring them into this discussion [kaospilot.dk](https://www.kaospilot.dk/). They've been going for over 30 years, and have tried to bridge the academic with entrepreneurial direct actions.

I've heard of kaospilot for a while. I'd love to know more especially about their curriculum.

> Without yet formulating a polished contribution, I'm drawn to using the Transcendentals to inform the design - what are the fundamental contrasting truths that the human condition is called to "coordinate" - i.e. The Good, True and Beautiful...The Morals, Sciences and Arts...

The "GTB" course then 😄

**Reply by rufuspollock** [reply]
@theo-cox i think you item is actually a new comment (not a response to @baouroux) so do you want to post as a root level comment so it can get voted on 😄

**Comment by Alex-Goodall** [comment]
Oh - very interesting, Rufus!

I'll add some ideas based on the draft syllabus of my http://Relevant.Education project.

It's not conceived of as a degree-level or style of education. Rather, the underlying premise is that we are heading for a transition, and our best chance of having it be a transformative rather than a dystopic transition is to have enough people with enough of a clue to be able to participate in meaningful conversations.

The "syllabus" is a big bunch of topics related to my ideas of what it means to have a clue. (Actually, the syllabus is an outline of what was to be a book, but it's become far too big, so it's now a draft syllabus for the Relevant.Education project as a whole.)

This sort of discussion is exactly what I need right now, so am very happy to get involved. Will post details later.

My acronym will be CRGOCP. So -20 points, I suppose.  

Alex

**Reply by baouroux** [reply]
Fabulous resources and work you've established here Alex! Just going back to your sites anew.

**Reply by Alex-Goodall** [reply]
Here's the draft syllabus for my Relevant Education project. Hopefully, there's something useful there for a PPE replacement: there's some good overlap with Rufuses re-definition of Psychology, Politics and Ecology.

MODULE ONE
Challenging the fundamentals

Topic 1 : I know nothing
Topic 2 : Worldviews and cultural indoctrination
Topic 3 : The Roles of Philosophy, Science and Religion
Topic 4 : Hemisphere-based knowing
Topic 5 : Being and Knowing
Topic 6 : Changing and Becoming
Topic 7 : Meaning and Purpose

MODULE TWO
Reviewing the past, examining the present

Topic 8a : Where we've come from – origin stories
Topic 8b : Where we've come from – multiple perspectives and explanations of history
Topic 9 : What we've done to ourselves: Crises of sense and meaning
Topic 10 : What we've created: Civilizational risks
Topic 11 : Delusions and dangers of current responses
Topic 12 : A Time Between Worlds

MODULE THREE
Growing, deciding, acting & flourishing

Topic 13 : Individual & Collective Growth
Topic 14 : Individual Growing, Deciding, Acting & Flourishing
Topic 15 : Collective Growing, Deciding, Acting & Flourishing
Topic 16 : Politics, Law and Governance
Topic 17 : Education


MODULE FOUR
Operating our Civilisation

This MODULE is something of an “everything else” catchall.
Each topic covers its dysfunctional aspects - and their implications - along with current activities and potentialities for how that topic could evolve to support a transformative transition.

Topic 18 : Money, Markets and Economics
Topic 19 : Geopolitics and Globalization
Topic 20 : Living & Working Environments
Topic 21 : Energy & Minerals
Topic 22 : Ecology and the Environment
Topic 23 : Health and Social Care
Topic 24 : Culture, Arts and Media
Topic 25 : Technology
Topic 26 : Food Production
Topic 27 : Artificial Intelligence: The disruptive driver of disruptors
Topic 28 : Digital, Biological and Materials Technology
Topic 29 : Transportation
Topic 30 : Hidden Power Dynamics and Sources

MODULE FIVE
Conceiving, gestating and birthing a transformed Humanity

Topic 31 : Levels of Humanity's transformation
Topic 32 : The Next Developmental Level
Topic 33 : The difficulty of describing what has yet to emerge
Topic 34 : Systematising Systemic Change and Enabling Emergence
Topic 35 : Ways of responding to the Metacrisis
Topic 36 : The Religious Instinct 3.0
Topic 37 : Future Forecasts of Key Sectors
Topic 38 : If the answer isn't Education, you're asking the wrong question
Topic 39 : Contours of a transformed Humanity
Topic 40 : Long-term perspectives


MODULE SIX
Participating in Humanity’s Transition

Topic 41 : Participate by Being & Knowing
Topic 42 : Participate by Becoming
Topic 43 : Participate by Doing
Topic 44 : Participate by Flourishing

**Reply by rufuspollock** [reply]
@Alex-Goodall just coming back to this and seeing this curriculum which looks great. Did you have content drafted for some of these sections?

**Comment by theo-cox** [comment]
**Repost**. Thanks for heads up @rufuspollock

For me - as a former PPEist who loved their degree but defnitely saw it's limitations, particularly around curriculum but also the pretty siloed approach to interdisciplinarity - it'd be less about setting out particular fields/disciplines (while of course necessary to some degree) but a different manner of teaching that goes further into transdisciplinary.

I'd want to see a course that gave open ended briefs to explore "big questions" (both at the level of how we organise our societies and how we see ourselves as humans etc), ideally with lots more working in groups. I'd want the course to steer students towards a holistic and integrative way of looking at and tackling such questions from multiple angles. Within reason (and while you'd of course need some scaffolding of knowledge etc) I'd even say it would be better to not specify the fields or intellectual traditions which should be used to encourage diversity and creativity.

I think Geoff Mulgan talks along these lines in his book on social and political imagination, which I definitely resonated with.

**Reply by rufuspollock** [reply]
What would your initial reading list be @theo-cox for people about to start the course?

What would they do in their first couple of terms?

**Reply by rufuspollock** [reply]
@theo-cox pinging again to see if you saw original comment. would be interesting to get more detail here.

**Comment by EmilEjnerFriis** [comment]
I've actually been thinking a lot about education these past days, and not least about my own role as a teacher. My thoughts have, however, revolved around children and adolescents. The term "leadership" made me think about all that so-called leadership training that you find in our circles, something I reckon is somewhat beyond my expertise given that my experience with working in teams, or telling other people what to do, is rather limited. When it comes to how we should reform our education system I could write a book about it, or at least a substantial paper -- which I'll refrain from doing in the thread.

Daniel and I are currently putting a mm course together. I'll gladly share it with you once we're done.

**Reply by Alex-Goodall** [reply]
Hi Emil

I too have a teacher's focus on children and adolescents. I was struck by the recent Hanzi article on [AI Apocalypse](https://medium.com/@hanzifreinacht/what-can-stop-the-ai-apocalypse-grammar-yes-only-grammar-fe6ab26cc30d) that ended with "I have said it before, and I’ll say it again: Our world will be conquered, ruled, and transfor­med by an army of children."

When I made my [presentation](https://www.youtube.com/watch?v=P0ZL-lFyXoc) about my project to LifeItself, the question I evoked was "How do we engage with curious mid-late teens to help prepare them to consciously participate in Humanity's Transition"

Alex

**Comment by ibearthling** [comment]
I am not born and bread British, but have heard of Oxford PPE degree.
It would be interesting to understand maybe why those graduates end up running the country?  Is it because they apply for certain jobs (which they are encouraged to do during the degree? ), because they believe that they are born to rule (which they are also encouraged to believe during their education)? is it because they join political parties, is it because the fame of the PPE is such that anybody who wants to compete with them feels it's better to simply roll over and surrender even before a proper 'fight'?    The 'queen's English that commands respect? The pomp? Any other reason?

Then the question could be asked: do we want the future UK to still be run by the graduates of the same college? Why?  If so, then the purpose of adapting the curriculum would seem to be: maintaining the status quo... would it not?

**Reply by rufuspollock** [reply]
Why Oxford PPE?

Because ... (these are my guesses / hypotheses)

- Oxbridge were the two elite universities and hence likely leaders are likely to go there
- Politicians incline to the humanities for various reasons ... and Oxford is stronger in that area (whilst Cambridge stronger in sciences)
- PPE is particularly suited for those going into politics as it gives a basic background in relevant areas

No particular suggestion that we want future UK to be run from the same college - and don't think that was the main thrust of the piece. (PPE point was a lead in to the discussion of what *should* a curriculum for leaders look like, wherever it was taught  ...)

**Comment by JayFliz** [comment]
Maybe it could be Psychology Ecology Philosophy Politics Economics 3.0

(Doughnut) Economics etc

Run it at Schumacher College.

**Reply by rufuspollock** [reply]
@JayFliz nice suggestion.

Out of interest what would you consider is in doughnut economics that is different from mainstream economics?

**Reply by JayFliz** [reply]
Maybe it is my experience and biases but I think of mainstream economics as very much the science of ever increasing consumption  ( and that is obviously very reductive) whereas the Doughnut is about a sustainable middle ground. 
Again, this may be my own ideology here but, I want economics to embrace the positives of capitalism ( improved housing and heating, the healthcare and ability to provide food - let's skip over the details of companies that make a profit from edible products ) whilst accepting that it needs to be sustainable.
Obviously that is not easy but we have to aim higher.

**Comment by Alex-Goodall** [comment]
I keep thinking about "what we need to know" - as leaders or "just" participants in the transition... and I keep coming back to the idea that, even more important than knowing stuff is the ability to loosen our knowing. Not hold it rigidly. To question our assumptions as deeply as possible. Down to metaphysical assumptions if I had my way.

I don't know how much Oxford's PPE is set up to do this or simply to pass on the received wisdom of the academic elders, but any replacement should include a massive dose of assumption questioning... which, of course, requires teachers who are not already indoctrinated.

**Reply by rufuspollock** [reply]
@Alex-Goodall so agree Alex and i would suggest that we could interpret "knowing" in the broad sense i.e. of skills and capacities rather than in the narrow (left hemisphere) sense of facts and procedures.

e.g. you could imagine most of this future course being transformational practices e.g. mindfulness, self inquiry etc.

**Comment by rufuspollock** [comment]
## Indian Gov National Education Policy 2020

https://www.education.gov.in/sites/upload_files/mhrd/files/NEP_Final_English_0.pdf

Someone just pointed me to this doc.

> 11.1. India has a long tradition of holistic and multidisciplinary learning, from universities such as
Takshashila and Nalanda, to the extensive literatures of India combining subjects across fields.
Ancient Indian literary works such as Banabhatta’s Kadambari described a good education as
knowledge of the 64 Kalaas or arts; and among these 64 ‘arts’ were not only subjects, such as
singing and painting, but also ‘scientific ’fields, such as chemistry and mathematics, ‘vocational ’
fields such as carpentry and clothes-making, ‘professional ’fields, such as medicine and engineering,
as well as ‘soft skills’ such as communication, discussion, and debate. The very idea that all
branches of creative human endeavour, including mathematics, science, vocational subjects,
professional subjects, and soft skills should be considered ‘arts’, has distinctly Indian origins. This
notion of a ‘knowledge of many arts’ or what in modern times is often called the ‘liberal arts’ (i.e., a
liberal notion of the arts) must be brought back to Indian education, as it is exactly the kind of
education that will be required for the 21st century.


> 21. Adult Education and Lifelong Learning

**Reply by JamieHBristow** [reply]
[Here's a Google doc](https://docs.google.com/document/d/1MWxd4i8XZ8OWn3T4-7w8ti8GiDw1pOdjRcWpOiX6V_I/edit?usp=sharing) with the main excerpts from India's National Education Policy 20202 relating to inner development within a vision for life-long holistic education.

**Comment by asimong** [comment]
Thanks for reviving this really interesting topic (disclosure: my degree was Physics and Philosophy at Oxford...)

Right now, for all the virtues of Oxford and Cambridge in terms of academic excellence, like @JayFliz I wouldn't run this from any traditional university, as I see them as either too stuck in tradition, or too much morphed into businesses to make money — competitively of course. While acknowledging and celebrating the great strides towards more inclusivity that have been taken, I'd say there's still an awful long way to go until we get even an intellectual education that is effectively open to all, let alone an emotional one.

The setting would therefore link up with our conversations on DDSs. (And I'd be particularly interested in what some of our newer people would say on this (such as Isabela, who I don't see here.)

Getting a good overall framework seems like a great idea, and I'm interested by the 1st-2nd-3rd person frame of @baouroux above.

Separately, I'd like to see the emphasis, not on learning one particular thread (though that's highly defensible) but on the recognition and understanding of divergent worldviews / personal ontologies, along with the psychological and sociological correlates of these. So, sure, some philosophy (not just the history of philosophy from Descartes to Kant!); some psychology, including cognitive biases, and the collective effects both good and bad, also anthropology and evolutionary psychology; some history (as e.g. Iain McGilchrist would argue), and certainly not focusing on kings and battles. I'd like to see some treatment of the interplay of the spiritual and the religious, right the way across philosophy, history, politics and psychology; and also a sound appreciation of the strengths and weaknesses of science and scientific method. This may be rather biassed, but I would like to see young people educated in the cognitive sciences, extending psychology to computational modelling and AI, along with its potential and its fundamental limitations (if you believe that!!!)

Recognising that I've set out too much to be covered properly in a three-year degree course, at a time of life when young adults are in any case in some kind of process of self-discovery, I'd like to see students forming strong coalitions where different individuals go deep into different specialisms, but keep talking with each other, cross-referencing and cross-pollinating the fruits of their study. We could see this also as embodied in a framework of collective study, collective intelligence, collective wisdom, beyond (rather than overlooking) the individualism of the "enlightenment", in order to foster a genuine humility of individual intellect.
